packages/techdocs-common/src/stages/publish/publish.test.ts

```diff
@@ -105,4 +106,28 @@ describe('Publisher', () => {
     });
     expect(publisher).toBeInstanceOf(AwsS3Publish);
   });
+
+  it('should create Azure Blob Storage publisher from config', async () => {
+    const mockConfig = new ConfigReader({
+      techdocs: {
+        requestUrl: 'http://localhost:7000',
+        publisher: {
+          type: 'azureBlobStorage',
+          azureBlobStorage: {
+            credentials: {
+              accountName: 'accountName',
+              accountKey: 'accountKey',
+            },
+            containerName: 'containerName',
+          },
+        },
+      },
+    });
+
```

Comment:
Is it possible to add another test where the account key is not defined and the environment variables are used?

This test could hijack the `process.env` and add some mock environment variables like [this](https://stackoverflow.com/questions/48033841/test-process-env-with-jest). Just exploring. :)